src/hotspot/share/opto/stringopts.cpp

```diff
@@ -173,6 +173,9 @@ class StringConcat : public ResourceObj {
     assert(!_control.contains(ctrl), "only push once");
```

Comment:
Remove the changes in this file.

Reply:
@afshin-zafari ,  here's one of the unhandled comments.

Reply:
Done.